MapleServer2/Packets/FieldPacket.cs

```diff
@@ -387,7 +388,7 @@ public static Packet RemoveMob(IFieldObject<Mob> mob)
         public static Packet AddPortal(IFieldObject<Portal> portal)
         {
             PacketWriter pWriter = PacketWriter.Of(SendOp.FIELD_PORTAL);
-            pWriter.WriteByte(0x00);
+            pWriter.WriteByte((byte) PortalType.AddPortal);
```

Comment:
Is PortalType being used anywhere outside this file? If not, you should put it in this life like other packet files do. Also you can use `WriteEnum` for this

Reply:
Oh, cool! I will put it in this file!